src/main/webapp/index-script.js

```diff
@@ -45,11 +46,15 @@ function loadMainButtons() {
   });
 
   commentSortRelevant.addEventListener("click", () => {
-    // TODO: Make sort relevant function
+    commentSortRelevant.classList.add("active");
+    commentSortRecent.classList.remove("active");
+    resortReviews(prev_ID, 'relevant');
   });
   
   commentSortRecent.addEventListener("click", () => {
-    // TODO: Make sort recent function
+    commentSortRecent.classList.add("active");
+    commentSortRelevant.classList.remove("active");
```

Comment:
Does this mean you're adding the button after they click on it? Shouldn't you be hiding the button that gets clicked on and revealing the opposite option instead?

Reply:
No this means that it adds the active styling for the button on click.
Activate styling just means that the button is now colored in a bright color to distinguish itself from the other.